week-9/Homework/mandatory/2-exercises/4-music-player.js

```diff
@@ -20,34 +20,50 @@ This means the order the songs are played in will be random, but each song will
 
 
 class MusicPlayer {
-    // Add your code here
+    constructor() {
+        this.playing = [];
+        this.MusicPlayer = [];
+        this.index = 0;
+    }
+    add(playing, player) {
+        this.playing.push(playing);
+        this.MusicPlayer.push(player);
+
```

Comment:
you're creating two arrays that are linked to each other (each song from array 1 will have a singer from array 2) to  exclude any issues, adding an array of object will be easier 
something like 
this.playList=[]
this.playlist.push({song:,  singer:})